[Bug]: E-Documents: Basic info from purchase credit memo: Order No. is not filled in

### Describe the issue

Hi,

In E-documents, when importing an purchase invoice the field "Order No. is filled with the tag "/Invoice/cac:OrderReference/cbc:ID".
When importing a purchase credit memo, this field stays empty.
See the procedures "ParseInvoiceBasicInfo" and "ParseCreditMemoBasicInfo" in codeunit "EDoc Import PEPPOL BIS 3.0" in E-Document Core.

Can this be fixed, please?

### Expected behavior

The "Order No." should be filled in when processing a purchase credit memo.

### Steps to reproduce

Process an e-document of a purchase credit memo with the tag cac:OrderReference in it.

### Additional context

_No response_

### I will provide a fix for a bug

- [x] #7747

Cloud you please approve this issue?
Pull request https://github.com/microsoft/BCApps/pull/7748 fails with the following error:

Exception: /home/runner/work/BCApps/BCApps/build/scripts/PullRequestValidation/ValidateIssuesForPullRequest.ps1:83
Line |
  83 |          throw "The following issues are not open or approved: $($inva …
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The following issues are not open or approved: 7746
Error: Process completed with exit code 1.

--- a/src/Apps/W1/EDocument/App/src/Format/EDocImportPEPPOLBIS30.Codeunit.al
+++ b/src/Apps/W1/EDocument/App/src/Format/EDocImportPEPPOLBIS30.Codeunit.al
@@ -104,6 +104,8 @@ codeunit 6166 "EDoc Import PEPPOL BIS 3.0"
         Evaluate(EDocument."Amount Excl. VAT", GetNodeByPath(TempXMLBuffer, RootPath + '/cac:LegalMonetaryTotal/cbc:TaxExclusiveAmount'), 9);
         Evaluate(EDocument."Amount Incl. VAT", GetNodeByPath(TempXMLBuffer, RootPath + '/cac:LegalMonetaryTotal/cbc:PayableAmount'), 9);
 
+        EDocument."Order No." := CopyStr(GetNodeByPath(TempXMLBuffer, RootPath + '/cac:OrderReference/cbc:ID'), 1, MaxStrLen(EDocument."Order No."));
+
         Currency := CopyStr(GetNodeByPath(TempXMLBuffer, RootPath + '/cbc:DocumentCurrencyCode'), 1, MaxStrLen(EDocument."Currency Code"));
         if LCYCode <> Currency then
             EDocument."Currency Code" := Currency;

